pg_search/src/postgres/types.rs

```diff
@@ -778,22 +778,48 @@ impl TryFrom<TantivyValue> for pgrx::JsonB {
     }
 }
 
+impl TryFrom<serde_json::Value> for TantivyValue {
+    type Error = TantivyValueError;
+
+    fn try_from(val: serde_json::Value) -> Result<Self, Self::Error> {
+        Ok(TantivyValue(tantivy::schema::OwnedValue::from(
+            serde_json::Value::Array(vec![val]),
+        )))
+    }
```

Comment:
This is a bit on an unexpected conversion: one direction wraps in an `Array`, while the other direction does not unwrap the `Array`.

It feels like the caller should do the array wrapping instead, and the conversion could assert that it's there (assuming it's needed)?

Reply:
Tantivy requires a json array at the top-level, even if it’s one value and even if that one value is an array. 

That’s why this blindly wraps “val” in an array. 

I’m not sure pushing that responsibility to the caller would be an improvement?